tests/test_warmer.py

```diff
@@ -47,8 +49,9 @@ def test_warmer_fan_out(self):
 
         self.assertFalse(LAMBDA_INFO['is_warm'])
 
-        self.decorared_dummy_lambda(self.warmer_invocation_event, get_context())
+        lambda_return_val = self.decorared_dummy_lambda(self.warmer_invocation_event, get_context())
 
+        self.assertIsNone(lambda_return_val)
```

Comment:
for a warmer invocation event, this will be `None` bc the [wrapped lambda function](https://github.com/robhowley/lambda-warmer-py/pull/10/files#diff-1b91f49ab6a0c193f582ae65d97694beR41) won't execute and return it's content